File: Omega/#2 Implementation/2-5.cpp
#include <bits/stdc++.h>

using namespace std;

string s;
int lef, rig;

int main() {
	cin >> s;
	int len = s.length();
	for (int i = 0; i < len; i++) {
		if (i < len / 2) {
			lef += s[i] - '0';
		}
		else {
			rig += s[i] - '0';
		}
	}
	if (lef == rig) {
		cout << "LUCKY" << "\n";
	}
	else {
		cout << "READY" << "\n";
	}
	return 0;
}

File: Omega/#2 Implementation/2-5.py
# 럭키 스트레이트

n = list(map(int, input()))
p = len(n) // 2
sum1 = 0
sum2 = 0

for i in range(0, p):
    sum1 += n[i]

for i in range(p, len(n)):
    sum2 += n[i]

if sum1 == sum2:
    print('LUCKY')
else:
    print('READY')